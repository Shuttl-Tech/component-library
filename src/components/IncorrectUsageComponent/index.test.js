import React from 'react';
import renderer from 'react-test-renderer';

import { IncorrectUsageComponent } from './';

it('IncorrectUsageComponent tree looks like how it should', () => {
	const component = renderer.create(<IncorrectUsageComponent/>);
	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
