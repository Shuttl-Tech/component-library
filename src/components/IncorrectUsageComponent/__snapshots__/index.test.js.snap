// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IncorrectUsageComponent tree looks like how it should 1`] = `
<div
  className="container"
>
  <div>
    <div>
      <span>
        <h3
          className="title"
        >
          Incorrect component usage.
        </h3>
        <small>
          You tried to use a default export from the
          <br />
          <strong>
            Shuttl Component Library
          </strong>
           as a component.
        </small>
      </span>
      <hr />
    </div>
  </div>
  <div>
    <span>
      Usage should be as follows:
    </span>
    <pre>
      <code>
        import { ComponentName } from 'component-library';
      </code>
      <br />
      <code>
        ...
      </code>
      <br />
      <code>
        &lt;ComponentName&gt;some content here&lt;/ComponentName&gt;
      </code>
    </pre>
  </div>
</div>
`;
